README.md

```diff
@@ -121,7 +121,46 @@ A style guide for the [HTML Template Language](https://docs.adobe.com/docs/en/ht
         ...
     </sly>
     ```
+    
+  - [3.4](#3.4) <a name='3.4'></a> **Avoid using the ternary operator unnecessarily.**
```

Comment:
I know that it's out of the scope in this PR and this is copied from Adobe's documentation, but Github automatically adds anchors when using the [header syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#headings).

Reply:
I didn't know that! I will open an issue to fix this, thanks!

Reply:
https://github.com/Netcentric/aem-htl-style-guide/issues/35